### PR TITLE
Double Pendulum: update double pendulum theory

### LINKED ISSUE #2618: Double Pendulum: Update the double pendulum example to actually be a double pendulum
The double pendulum example only presents the theory for one pendulum.  We should update it for two rods.

### PR DESCRIPTION
**Pull request by cd155**
close #2618

@smiths The theory of double pendulum has been updated, it is mainly based on 
https://www.myphysicslab.com/pendulum/double-pendulum-en.html

In the end, it concludes two equations θ1'', θ2'' in terms of θ1, θ1', θ2, θ2'

It is my first draft, if there is anything you think I should remove or include, please let me know.

These are links for this change
https://github.com/JacquesCarette/Drasil/blob/updateDblPendulumThoery/code/stable/dblpendulum/SRS/PDF/DblPendulum_SRS.tex
https://github.com/JacquesCarette/Drasil/blob/updateDblPendulumThoery/code/stable/dblpendulum/SRS/HTML/DblPendulum_SRS.html

### CODE REVIEW

File: code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Assumptions.hs [hunk 1]
```diff
@@ -1,40 +1,55 @@
...
 
-pend2DMotion, cartCoord, cartCoordRight, yAxisDir, startOrigin :: ConceptInstance 
+twoDMotion, cartSys, cartSysR, yAxisDir, startOrigin, firstPend, secondPend:: ConceptInstance 
 
-pend2DMotion    = cic "pend2DMotion"      pend2DMotionDesc    "pend2DMotion"    assumpDom
-cartCoord       = cic "cartCoord"         cartCoordDesc       "cartCoord"       assumpDom
-cartCoordRight  = cic "cartCoordRight"    cartCoordRightDesc  "cartCoordRight"  assumpDom
-yAxisDir        = cic "yAxisDir"          yAxisDirDesc        "yAxisDir"        assumpDom
-startOrigin     = cic "startOrigin"       startOriginDesc     "startOrigin"     assumpDom
+twoDMotion  = cic "twoDMotion"      twoDMotionDesc    "twoDMotion"    assumpDom
```
Comment (JacquesCarette) [inline code/drasil-example/dblpendulum/lib/Drasil/DblPendulum/Assumptions.hs:21]: all the columns used to be nicely lined up - can you line them up again please?

**Review by JacquesCarette — CHANGES_REQUESTED** [review]
Just one tiny change to make, and this is good to go.

**Comment by JacquesCarette** [comment]
and a master merge too.

**Comment by cd155** [comment]
@JacquesCarette Yes, those codes should be lined up now.

**Review by JacquesCarette — APPROVED** [review]

**Comment by JacquesCarette** [comment]
Great, thanks. Except that now that I've merged more things, it needs another master merge!

**Review by JacquesCarette — APPROVED** [review]

**Pull request merged**